.circleci/config.yml

```diff
@@ -56,7 +62,16 @@ build_test: &build_test
     name: "Build and test"
     command: |
         ./gradlew clean
-        ./gradlew build --debug
+        ./gradlew build --debug -x :android_instrumented_test:build
+
+build_test_android: &build_test_android
+  run:
+    name: "Build and test"
+    no_output_timeout: 20m
+    command: |
+      ./gradlew clean
+      ./gradlew connectedAndroidTest
+      ./gradlew build --debug -x :android_instrumented_test:build -x test
```

Comment:
Why did you build a source after test?

Reply:
@sirano11 
I adjusted the order of steps as described in the name of the step :)

F.Y.I
Here is the description about gradle commands about `android_instrumented_test` sub-project.
```
Build tasks
-----------
android_instrumented_test:build - Assembles and tests this project.
android_instrumented_test:connectedDebugAndroidTest - Installs and runs the tests for debug on connected devices.
```